apps/client/src/app/pages/portfolio/activities/import-activities-dialog/import-activities-dialog.html

```diff
@@ -4,112 +4,122 @@
   [title]="errorMessages.length === 0 ?  'Import Activities' : 'Import Activities Error'"
   (closeButtonClicked)="onCancel()"
 ></gf-dialog-header>
+<link
```

Comment:
You cannot do it like this 😄 But it's easy to use icons, see the example [here].(https://github.com/ghostfolio/ghostfolio/blob/main/libs/ui/src/lib/trend-indicator/trend-indicator.component.html#L12). 

The `name` you can find in this icon set: https://ionic.io/ionicons